ParisBNF/abb/BNFabb51.xml

```diff
@@ -0,0 +1,495 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb51" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title></title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt>
+                <p/>
+            </editionStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100878602.r=Abbadie%2051?rk=64378;0">BnF Éthiopien d'Abbadie 51</idno>
+                  </msIdentifier>
+
+                    <msContents>
+                      <msItem xml:id="ms_i1">
+                        <locus from="1ra" to="77vb"/>
+                         <title type="complete" ref="LIT2290Sargis"></title>
+                         <textLang mainLang="gez"/>
+                         <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወል</hi>ድ፡ ወመንፈስ፡ ቅዱ<hi rend="rubric">ስ፡
+                           ፩፡ አምላክ፡ ፍቁራንየ፡</hi> ስምዑ፡ ዘ<hi rend="rubric">ኮነ፡ በመዋዕለ፡ ሀ</hi>ርቃል፡ ንጉሠ፡ ሮ<hi rend="rubric">ም፡ በእንተ፡ ብእ</hi>ሲ፡
+                           ዘስሙ፡ ሰርጊ<hi rend="rubric">ስ፡ አብርጋዊ፡ ዘእ</hi>ምሥራዊተ፡ ንጉሥ፡ ናሁ፡ ሤሞ፡ መስፍነ፡ ላዕለ፡ ክልኤ፡ አህጉር፡ አፍራጊያ፡
+                         </incipit>
+                        </msItem>
+
+                         <msItem xml:id="ms_i2">
```

Comment:
can it be that there is a change in msPart? new text, new quire, new layout, new hand?

Reply:
as above.